spec/watirspec/elements/button_spec.rb

```diff
@@ -75,6 +75,23 @@
       expect(browser.button(xpath: "//input[@type='button']")).to exist
     end
 
+    it "matches the specific type when locating by type" do
+      aggregate_failures do
```

Comment:
I'm all for consistency, but if that's the only opposition, would it be better to start switching the existing specs to use aggregate_failures? It just seems weird that if we have a number of independent checks that we have to work through one failure at a time rather than knowing all of the failures.